packages/client/modules/teamDashboard/components/AgendaList/AgendaList.tsx

```diff
@@ -58,8 +58,8 @@ const AgendaList = (props: Props) => {
     ) {
       return
     }
-    const sourceItem = agendaItems[source.index]
-    const destinationItem = agendaItems[destination.index]
+    const sourceItem = agendaItems[source.index]!
+    const destinationItem = agendaItems[destination.index]!
```

Comment:
please don't assert non-null-ness if it's not clear from the immediate code before, that it's not null.